jwst/gain_scale/gain_scale.py

```diff
@@ -13,29 +10,40 @@ def do_correction(input_model, gain_factor):
     Short Summary
     -------------
     Rescales all integrations in an exposure by gain_factor, to
-    account for non-standard detector gain settings. The SCI
-    and ERR arrays are rescaled.
+    account for non-standard detector gain settings. The SCI,
+    ERR, and variance arrays are rescaled.
 
     Parameters
     ----------
-    input_model: data model object
-        science data to be corrected
+    input_model : `~jwst.datamodels.DataModel`
+        Input datamodel to be corrected
 
     Returns
     -------
-    output_model: data model object
-        rescaled science data
+    output_model : `~jwst.datamodels.DataModel`
+        Output datamodel with rescaled data
 
     """
 
     # Create output as a copy of the input science data model
     output_model = input_model.copy()
 
-    # Apply the rescaling to the entire data array
+    # Apply the gain factor to the SCI and ERR arrays
```

Comment:
Perhaps:
 # Apply the gain factor to the SCI, ERR, and variance arrays.

Reply:
Ignore my comment, as you mentioned it in the later comment (more appropriately)